File: 251001/Levitskij/DistComp/module-publisher/src/main/java/by/kopvzakone/distcomp/dto/EditorRequestTo.java
package by.kopvzakone.distcomp.dto;

import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class EditorRequestTo {
    long id;

    @Size(min = 2, max = 64)
    String login;

    @Size(min = 8, max = 128)
    String password;

    @Size(min = 2, max = 64)
    String firstname;

    @Size(min = 2, max = 64)
    String lastname;
}

File: 351002/Yanachkin/Lab6/lab1/Models/DTO/Requests/EditorRequestTo.cs
namespace lab1.Models.DTO.Requests
{
    public class EditorRequestTo
    {
        public long Id { get; set; }
        public string Login { get; set; } = null!;
        public string Password { get; set; } = null!;
        public string Firstname { get; set; } = null!;
        public string Lastname { get; set; } = null!;
        public string? Role { get; set; }
    }
}
